thorlcr/activities/diskread/thdiskreadslave.cpp

```diff
@@ -1197,6 +1197,7 @@ class CDiskGroupAggregateSlave
 
                 if (!container.queryLocalOrGrouped() && container.queryJob().querySlaves()>1)
                 {
+                    BlockedActivityTimer t(totalCycles, timeActivities);
```

Comment:
unfortunately, it's not as easy as this..
This will not block/will complete quickly, no matter what the data size.

aggregateStream object will be returned quickly, but calling it (nextRow) will cause blocking as it reads from the distributor inside the implementation (CAggregatingStream).
So the blocked time needs to be tracked there.